exercises/5_account_balance.rb

```diff
@@ -0,0 +1,45 @@
+class Account
+  attr_reader :name, :account_no, :balance
+  attr_accessor :name, :balance
```

Comment:
Do we have to put name, balance in attr_reader also if we are putting it in attr_accessor..? Do read it once

Reply:
I will remove the attr_reader statement